roles/inspec/tasks/main.yml

```diff
@@ -10,18 +10,39 @@
     mode: 0755
   with_items:
     - /etc/inspec
-    - /etc/inspec/controls
-    - /etc/inspec/libraries
+    - /etc/inspec/host-controls
+    - /etc/inspec/host-controls/controls
+    - /etc/inspec/host-controls/libraries
 
-- name: inspec config file
+- name: download and unpackage dependency profiles
```

Comment:
we usually stage the file and then unarchive it.   you also probably want to version it and symlink it in, otherwise when the contents of the archive change you'll see some really weird stuff.    check out how the monitor tarball is handled - https://github.com/blueboxgroup/ursula/blob/master/roles/common/tasks/monitoring.yml#L117-L137